newdle/core/util.py

```diff
@@ -89,3 +90,10 @@ def find_overlap(day, start, end, tz):
 def dedent(value, *, _re=re.compile(r'^ +', re.MULTILINE)):
     """Remove leading whitespace from each line."""
     return _re.sub('', value)
+
+
+def render_user_avatar(user, size):
+    avatar = render_template(
+        'avatar.svg', text=user['first_name'][0].upper(), size=size
+    )
+    return send_file(BytesIO(avatar.encode('utf-8')), mimetype='image/svg+xml')
```

Comment:
`.encode()` should be enough in python 3

Reply:
Oh.. true! I forgot to remove it.